plone/app/contenttypes/tests/test_webdav.py

```diff
@@ -0,0 +1,56 @@
+import os.path
+from plone.app.contenttypes.testing import (
+    PLONE_APP_CONTENTTYPES_INTEGRATION_TESTING,
+)
+from plone.app.testing import TEST_USER_ID, setRoles
+import unittest2 as unittest
+from zope.publisher.browser import TestRequest
+from ZPublisher.HTTPResponse import HTTPResponse
```

Comment:
Case insensitive sorted imports, please. 
You may use ``isort``, which is really handy (here is also a little section about isort: http://docs.plone.org/develop/styleguide/python.html#about-imports )